Extensions/xkit_patches.js

```diff
@@ -180,9 +180,146 @@ XKit.extensions.xkit_patches = new Object({
 		}, 1000);
 	},
 
-	run_order: ["7.8.1", "7.8.2", "7.9.0", "7.9.1", "7.9.2"],
+	run_order: ["7.8.1", "7.8.2", "7.9.0", "7.9.1", "7.9.2", "7.9.3"],
 
 	patches: {
+		"7.9.3": function() {
```

Comment:
there is no XKit 7.9.3, this should be in the 7.9.2 function

Reply:
Updated!